_posts/2020-04-23-putting-the-serverless-in-graphQL-with-AppSync.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Putting the Serverless in GraphQL with AppSync
+date: 2020-04-23
+tags: AWS AppSync Cloudformation
+author: Shermayne Lee
+---
+
+# Overview
+
+AWS AppSync and GraphQL have gained popularity in recent years. However it can be confusing and difficult to start for developers new to GraphQL concept.
+
+GraphQL is an open source query language that was developed to allow communication between your application and server. For more information visit [GraphQL](https://GraphQL.org/).
+
+AWS AppSync is a managed, serverless, GraphQL implementation on AWS. It can translate GraphQL requests from client, get the data from AWS Resources and pass the data to the client.
```

Comment:
> It can translate GraphQL requests from client, get the data from AWS Resources and pass the data to the client.

It can translate GraphQL requests from a client

This sentence sounds a bit weird though - from client to client?